website/docs/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -348,9 +348,9 @@ This permission confers all of the permissions granted by the manage teams and m
 
 ~> This permission is intended to allow owners of large organizations to delegate further team management control to another trusted team, and should be granted to only teams of trusted users. **Assign with caution:** Members with this permission will be able to update all organization access settings for any team visible to them.
```

Comment:
Non-Blocking Nitpick: I think that the "to" should be after the "only" here, but I could be mistaken!:
```suggestion
~> This permission is intended to allow owners of large organizations to delegate further team management control to another trusted team, and should be granted only to teams of trusted users. **Assign with caution:** Members with this permission will be able to update all organization access settings for any team visible to them.
```